docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -740,6 +740,45 @@ date.in_time_zone
 | `strict`, `flexible`
 |===
 
+== Rails/DefaultScope
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
```

Comment:
Please don't enable it by default. It's problematic, but there is a correct way to use it.

Reply:
If I'm not mistaken, `Enabled: pending` does not enable it but gives user a choice to disable or enable it.
And finally `Enabled: pending` will be changed to `Enabled: false` by default.

Reply:
According to the [Versioning policy](https://github.com/rubocop-hq/rubocop/blob/master/docs/modules/ROOT/pages/versioning.adoc#pending-cops):

> On major version updates, pending cops are enabled in bulk

Reply:
The pending status feature is not to advertise the new cops that are introduced between major releases, it's to relieve the pain of cops being introduced with "Enabled" by default causing red CI for those who keep their code offence-free.

New cops, including those disabled by default, will be advertised somehow differently, probably in a post-install message.

Reply:
Ah, thanks for explanation. Made it disabled by default.